google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcStorageImpl.java

```diff
@@ -830,7 +834,7 @@ public BlobInfo internalDirectUpload(
     requireNonNull(blobInfo, "blobInfo must be non null");
     requireNonNull(buf, "content must be non null");
     OpenTelemetryTraceUtil.Span otelSpan =
-        openTelemetryTraceUtil.startSpan("internalDirectUpload(BlobInfo)", ctx);
+        openTelemetryTraceUtil.startSpan("internalDirectUpload(BlobInfo)", OTEL_MODULE_NAME, ctx);
```

Comment:
method name has parameters when not needed

Reply:
ops. missed that one when removing parameters. fixed.